test/sqllogictest/decimal.slt

```diff
@@ -240,14 +240,53 @@ SELECT 1 * 1 * 1 * 1 * 1 * 1 * 1 * 1 * 1 * 1 * 1.0
 ----
 1
 
+### edge cases ###
+query error invalid input syntax for numeric: malformed numeric literal: : ""
+SELECT ''::decimal
+
+query R
+SELECT '  3  '::decimal
+----
+3
+
+query R
+SELECT '99999999999999999999999999999999999999'::decimal
+----
+99999999999999999999999999999999999999
+
+query R
+SELECT '-99999999999999999999999999999999999999'::decimal
+----
+-99999999999999999999999999999999999999
+
+query error exceeds maximum decimal precision
+SELECT '100000000000000000000000000000000000000'::decimal
+----
+100000000000000000000000000000000000000
+
+query error exceeds maximum decimal precision
+SELECT '100000000000000000000000000000000000000'::decimal
+----
+100000000000000000000000000000000000000
```

Comment:
Is this just the same as the one above? Maybe meant to have a negative sign?

Reply:
Oops. Thanks!